certificate-policy.md

```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 
 ## 1.5 Policy administration
-This Certificate Policy for the Issuance and Management of Publicly-Trusted Certificates includes criteria established by the CA/Browser Forum for use by Certification Authorities when issuing, maintaining, and revoking publicly-trusted Certificates. This Certificate Policy also includes criteria established by the U.S. Federal Public Key Infrastructure to comply with U.S. Federal Government requirements for U.S. Federal Government Agencies.   This CP may be revised from time to time, as appropriate, in accordance with procedures adopted by the CA/Browser Forum and/or the Federal Public Key Infrastructure.
-
+The Federal Public Key Infrastructure Policy Authority (FPKIPA) manages this policy and represents the interests of the  U.S. Federal CIOs and CISOs.   
 
 ### 1.5.1 Organization Administering the Document
-The Federal Public Key Infrastructure Policy Authority (FPKIPA) is a group of U.S. Federal Government Agencies and is chartered by the U.S. Federal CIO Council.  The FPKIPA owns this policy and represents the interest of the Federal CIOs.  The FPKIPA is responsible for:  
+The Federal Public Key Infrastructure Policy Authority (FPKIPA) is responsible for:  
 * Maintaining this CP, and
 * Approving the CPS for each CA that issues certificates under this policy, and
 * Approving the compliance audit report for each CA issuing certificates under this policy, and
 * Ensuring continued conformance of each CA that issues certificates under this policy with applicable requirements as a condition for allowing continued participation, and
 * Ensuring compliance with CAB Forum Baseline Requirements
 
+The Federal PKI Policy Authority SHALL review and update this Certificate Policy at least every 365 days to ensure compliance with CAB Forum Baseline requirements.  After review and approval, the document version number and a dated changelog entry shall be added even if no changes were deemed necessary. 
+
+An annual self-assessment shall be conducted by any CA operating against this Certificate Policy and the accompanying CPS to show compliance with the latest version of the CAB Forum Baseline Requirements. The CA shall indicate the self-assessment annually by incrementing the document version number and adding a dated changelog entry to the change record. 
 
 ### 1.5.2 Contact Person
-Contact information for the Federal Public Key Infrastructure Policy Authority is **TODO: Insert Contact Info**.
+Contact information for the Federal Public Key Infrastructure Policy Authority is fpki@gsa.gov.
```

Comment:
Suggest adding a statement in Section 1.0 Overview to the effect that this policy incorporates Certificate Transparency as a key component in providing a publicly accessible and accountable PKI.
This sets the stage up front and provides an anchor for additional references to CT later in the policy.